docs/src/cli/examples/delegate-stake.md

```diff
@@ -50,6 +50,8 @@ solana create-stake-account --from <KEYPAIR> stake-account.json <AMOUNT> \
 `<AMOUNT>` tokens are transferred from the account at the "from" `<KEYPAIR>` to
 a new stake account at the public key of stake-account.json.
 
+Instead of a numeric amount, you can also use the keyword `ALL` to send all available SOL from the `--from` account, leaving only the minimum rent-exempt balance.
```

Comment:
I don't think this is correct, using `ALL` will completely drain the source account for system accounts. I didn't dig in too much more, but is there a situation in which the source account is left intact?